BREAKING.md

```diff
@@ -4,6 +4,8 @@
 - [documentId removed from IFluidDataStoreRuntime and IFluidDataStoreContext](#documentId-removed-from-IFluidDataStoreRuntime-and-IFluidDataStoreContext)
 - [@fluid-experimental/tinylicious-client package name changed](#fluid-experimentaltinylicious-client-package-name-changed)
 - [@fluid-experimental/fluid-static package name changed](#fluid-experimentalfluid-static-package-name-changed)
+- [FlushMode enum values renamed](#FlushMode-enum-values-renamed)
```

Comment:
I think it's great to have it as a single change to experiment, but I'd suggest submitting it in staging:
1. Change in naming (simple, unless we find it crosses compat layers)
2. Change in default batching behavior
3. Removal of batch manager.

Reply:
1&2 are in https://github.com/microsoft/FluidFramework/pull/7071, although I see it breaks a test (starting today after merging in main).

Reply:
Also I'd suggest flighting # 2. I.e. ship it dark and have some overwrite (localStorage for lack of better tools) to enable it in Bohemia and see what breaks. We (I :)) might be underestimating impact and the need to turn it off to give people enough time to address bugs that likely were always there but were hidden / less likely to be hit.

Reply:
I agree with this approach and will do that. First I need to merge https://github.com/microsoft/FluidFramework/pull/7071 after I figure out why the `todo item` tests started failing intermittently. This PR also saw the test failures but they cleared after re-running. I want to first make sure I'm not adding instability with the flushmode change..